src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -27,17 +30,24 @@ public class HttpProtocolTests : HttpClientTest
         [InlineData("HTTP/1.1 500 Internal Server Error", 500, "Internal Server Error")]
         [InlineData("HTTP/1.1 555 we just don't like you", 555, "we just don't like you")]
         [InlineData("HTTP/1.1 600 still valid", 600, "still valid")]
-        // TODO #24713: The following pass on Windows on .NET Core but fail on .NET Framework.
-        //[InlineData("HTTP/1.1 200      ", 200, "")]
-        //[InlineData("HTTP/1.1 200      Something", 200, "Something")]
-        //[InlineData("HTTP/1.1\t200 OK", 200, "OK")]
-        //[InlineData("HTTP/1.1 200\tOK", 200, "OK")]
-        //[InlineData("HTTP/1.1 200", 200, "")]
-        //[InlineData("HTTP/1.1 200\t", 200, "")]
-        //[InlineData("HTTP/1.1 200 O\tK", 200, "O\tK")]
-        //[InlineData("HTTP/1.1 200 O    \t\t  \t\t\t\t  \t K", 200, "O    \t\t  \t\t\t\t  \t K")]
-        //[InlineData("HTTP/1.1 999 this\ttoo\t", 999, "this\ttoo\t")]
         public async Task GetAsync_ExpectedStatusCodeAndReason_Success(string statusLine, int expectedStatusCode, string expectedReason)
+        {
+            await GetAsyncSuccessHelper(statusLine, expectedStatusCode, expectedReason);
+        }
+
+        [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "The following pass on .NET Core but fail on .NET Framework.")]
+        [InlineData("HTTP/1.1 200", 200, "")] // This test data requires the fix in .NET Framework 4.7.3
```

Comment:
The SkipOnTargetFramework attribute will make us to skip all 3 test cases on Framework. Is that what you intended? Should all 3 test cases be disabled on .NET Framework, or just the 1st one?

Reply:
All three test cases should be disabled against Framework. Because on .NET Framework is more strict with then content of status description (which is a bug in Framework, `reason-phrase  = *( HTAB / SP / VCHAR / obs-text )`, `\t` should be allowed in status description).

Reply:
LGTM then.